app/src/views/components/friends/list_of_filtered_users.tsx

```diff
@@ -29,7 +28,7 @@ export default function ListOfFilteredUsers({
           data={filteredUsers}
           keyExtractor={(item) => item.uid || Math.random().toString()}
           renderItem={({ item }) => {
-            const { isFriend, isRequested } = userStatuses[item.uid] || {
+            const { isFriendB, isRequestedB } = userStatuses[item.uid] || {
```

Comment:
Same here, this should be renamed while still making sure to not conflict with the Firestore one.